main/acle.md

```diff
@@ -7156,6 +7158,26 @@ Assuming `float64_t` data, the C version of the code above would be:
   p3 = svrdffr();
 ```
 
+### SVE reinterpret intrinsics
+
+The `svreinterpret` intrinsics for vector types and vector tuple types take the form
+
+``` c
+svreinterpret_type0[_type1][_xN]
+```
+
+where `N` refers to the number of elements in a tuple type.
+
+For example:
+``` c
+svuin16_t svreinterpret_u16_s32(svint32_t op);
+svuin16_t svreinterpret_u16(svint32_t op);
+svuin16x2_t svreinterpret_u16_s32_x2(svint32x2_t op);
+svuin16x2_t svreinterpret_u16(svint32x2_t op); 
+```
+
+A list of SVE reinterpret intrinsics can be found on [developer.arm.com](https://developer.arm.com/architectures/instruction-sets/intrinsics/#q=svreinterpret).
+
 ### List of SVE intrinsics
 
 The full list of SVE intrinsics can be found on
```

Comment:
I think Richard said to remove "full" form this sentence too:
The list of SVE intrinsics can be found on